scope & closures/ch5.md

```diff
@@ -133,7 +133,7 @@ foo();
 bar(); // 2
 ```
 
-Whatever facility we use to *transport* an inner function outside of its lexical scope, it will maintain a scope reference to where it was originally declared, and wherever we execute it, that closure will be exercised.
+Qualquer forma que usarmos para *transportar* uma função interna para fora do seu escopo léxico, ela irá manter uma referência de escopo de onde ela for declarada originalmente, e onde for que a executarmos, essa closure será utilizada.
 
```

Comment:
-"~~Qualquer~~ **Seja qual for a** forma que usarmos para *transportar* uma função interna para fora do seu escopo léxico, ela irá manter uma referência de escopo de onde ela for declarada originalmente, e onde for que a executarmos, essa closure ~~será utilizada~~ **ocorerá**."
 - A palavra *ocorrerá* pode ser trocada por **irá ocorrer**, **será exercidata** ou **será exercida**.
 - Acho que *facility* está com a ideia de **facilidade**, algo que facilite o processo. De qualquer maneira, acredito que a palavra *forma* também transmita a ideia. http://www.thefreedictionary.com/facility